.github/workflows/Invalidations.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     steps:
     - uses: julia-actions/setup-julia@v1
       with:
-        version: '1'
+        version: '1.10'
```

Comment:
same for invalidations here

Reply:
Invalidations are broken in 1.11 at the moment